__tests__/news.test.js

```diff
@@ -465,4 +465,52 @@ describe('GET /api/users', () => {
     });
 });
 
+describe('GET /api/articles topic query', () => {
+    test('should return specified topic from articles', () => {
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then((topic) => {
+            const topicsArray = topic.body.article[0]
+            expect(topicsArray).toMatchObject({
+                article_id: 5,
+                title: "UNCOVERED: catspiracy to bring down democracy",
+                topic: "cats",
+                author: "rogersop",
+                created_at: expect.any(String),
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              })
```

Comment:
We will need to check that all of the articles are about cats, not just the first :)